webrender/src/clip_scroll_node.rs

```diff
@@ -117,9 +145,16 @@ impl ClipScrollNode {
     }
 
     pub fn update_transform(&mut self,
-                            parent_world_transform: &ScrollToWorldTransform,
-                            parent_viewport_rect: &ScrollLayerRect) {
-        let inv_transform = match self.local_transform.inverse() {
+                            parent_reference_frame_transform: &LayerToWorldTransform,
+                            parent_combined_viewport_rect: &ScrollLayerRect,
+                            parent_accumulated_scroll_offset: LayerPoint) {
+
+        let local_transform = match self.node_type {
+            NodeType::ReferenceFrame(transform) => transform,
+            NodeType::ClipRect => LayerToScrollTransform::identity(),
```

Comment:
The offset and transform are combined into the self.world_viewport_transform below. Perhaps I'm not answering the right question though?